collectcontexts.py

```diff
@@ -32,40 +32,70 @@ def gather_chapter(api, book, chap):
         node = next_verse[0]
     return nodes
 
-def gather_book(api, book):
+def gather_book(api, book, lang):
     global VERSE_NODES
     result = dict()
     chap = 1
-    VERSE_NODES[book] = dict()
+    VERSE_NODES[lang][book] = dict()
     while True:
-        nodes = gather_chapter(api, book, chap)
+        nodes = gather_chapter(api, book, chap, lang)
         if nodes is None:
             return result
         result[chap] = nodes
         chap += 1
 
-def dump_book(api, book):
-    nodesets = gather_book(api, book)
+def dump_book(api, book, lang, use_features):
```

Comment:
- Please swap `book` and `lang`.
- Would it be possible to make `FEATURES` a dictionary just like `VERSE_NODES` so that you can use `FEATURES[lang]` and don't have to pass `use_features`?